### ISSUE TITLE
latest `redis` dependency update breaks New Relic instrumentation

### ISSUE THREAD

**Issue by ebr** [joshuah/sol-redis-pool#15]
This may be a New Relic - specific issue, but i wanted to flag it here just in case / as an FYI.

our database metrics in NR are broken with the latest `redis` npm module update.

we are using the `cache-manager-redis` npm module in our app, which depends on `sol-redis-pool`.

https://github.com/joshuah/sol-redis-pool/blob/master/package.json#L22 causes the `redis` dependency to be updated to `v2.6.0` as of yesterday, and that breaks things.

```
├─┬ cache-manager-redis@0.2.2
│ └─┬ sol-redis-pool@0.2.4
│   └─┬ redis@2.6.0
│     ├── redis-commands@1.2.0
│     └── redis-parser@2.0.0
```

**Comment by ebr** [comment]
update: the latest `redis` module (2.6.1 - released 20 minutes ago) fixes this. it was exporting an incorrect function name, which is now fixed.
still, may be a good idea to lock the dependencies to a point release to avoid this in the future.

**Comment by joshuah** [comment]
Lol, they kept updating the module faster than I had time to update it.

**Comment by ebr** [comment]
Ugh. Spoke too soon.
the latest update DOES break New Relic - i was just too tired last night when trying `redis@2.6.1`, and likely was actually testing `2.5.3`.
(I have an issue open with New Relic about this, but perhaps you'd want to lock it down anyway for now)

**Comment by joshuah** [comment]
Which invalid function name was being imported?

**Comment by ebr** [comment]
I assumed it was caused by this: https://github.com/NodeRedis/node_redis/issues/1075, but it's likely something else, since that's fixed in `2.6.1`. All I know for now is `2.6.1` breaks and `2.5.3` works, but haven't tested it much further yet.

**Comment by joshuah** [comment]
Set the minimal version to 2.6.2 in the 0.3.0 version.
